Improve explanations in getting started tutorial (#9299)
wagtail/wagtail: docs/getting_started/index.md

## Quick install

Run the following commands in a virtual environment of your choice:

```sh
$ pip install wagtail
```

> (Installing wagtail outside a virtual environment may require `sudo`. sudo is a program for unix-like computer operating system that enables users to run programs with the security privileges of another user, by default the superuser)

Reviewer: ```suggestion
(Installing wagtail outside a virtual environment may require `sudo`. sudo is a program to run other programs with the security privileges of another user, by default the superuser)
```

```sh
$ pip install -r requirements.txt
$ python manage.py migrate
$ python manage.py createsuperuser

> $ python manage.py runserver

Reviewer: I was surprised by those changes, as this shouldn’t be needed (the file already has a shebang to determine what to run it with). But,

- On our "try Wagtail in 7 lines" on https://wagtail.org/, we do use `python manage.py`, and we’ve researched this a lot – so I assume it’s very likely to work in lots of places
- Django’s tutorial as well uses `python manage.py` (https://docs.djangoproject.com/en/4.1/intro/tutorial01/).

So – great!
